### PR TITLE
Replace kafka functionality

### PR DESCRIPTION
**Pull request by ghost**
added endpoint properties

### CODE REVIEW

File: profiles/profiles.yml [hunk 1]
```diff
@@ -94,6 +94,12 @@ health:
 sentry:
   dsn: https://d2c258aaf0194406a4b684d5671b8f03@o1167145.ingest.sentry.io/6257927
 
+gamification:
+  url: http://localhost:8080
+  save-user-data: api/v1/process-user-data/SAVE_USER_DATA
+  remove-user-tasks: api/v1/process-user-data/REMOVE_USER_TASKS?userId={userId}
+
+
```
Comment (ghost) [inline profiles/profiles.yml]:
```suggestion
events:
  user-profile:
    gamification:
      url: http://localhost:8081
      endpoints:
        save-user-data: /api/v1/user-data/save
        remove-user-tasks: /api/v1/user-data/tasks/remove?userId={userId}
```

File: profiles/profiles.yml [hunk 2]
```diff
@@ -94,6 +94,12 @@ health:
...
```
Comment (ghost) [inline profiles/profiles.yml]: leave only one empty line at EOF

File: profiles/profiles-kubernetes.yml [hunk 3]
```diff
@@ -81,3 +81,8 @@ program:
 calendly:
   organization-id: 9b1093f4-7238-45bb-82f1-cd7d3278bf04
   key: '{cipher}0d51e0bef2641c5e5be7a2bad3b29d4c9d8844de0587ab0b5b91d02428322ca905781969ff8397edc23a441a0f57803da968f0b05df1a338b82e33a94f96f31d137941ff7281ed6af6415ec3c48e2abb94518d200f2d63e28c48434497ea05b33eec4ac643897f3ea81adfaab8082667457fec4c3ad725de225d1fdcca93607021b1eaa112be909b6a615820597b20d3c95e33b79ed0c2ea1282030c491fed0efbd481471ffaeb6b854d2d73b8aceae7ef77c7f41946354484457933a3837914dac36db76a0eba72ef8ca238578a67f1e11ff63e4bb0f67c019de731d01d36fdd5b520c7751f5efb5a7dd25d532c5eb96a867228a1317c1fee712fec74918866e5172c5b95475db6c6863171be7a50f8481196afb4b773b7f535c59ec04a773989771950b01361d56e706bd46c50067e186be0fd68fddc3bcc2313fff4a88f392b57357140fa876715a238fbabf9d25c6a4638bd7c62c7153fb8e2976b121449cb865714b2096066dbb85a2d31861a90ae0e84679ed08151860b828adb4fe95d73f8cb6ecc0bec965da86fb795ef0d0e721a39818653b8d3660014b46f4695595945f63fc361d67ec0bda459efbdfef4dfa8f0a0d01d8e57105be1c09f17a4f0'
+
+gamification:
+  url: http://gamification-manager
+  save-user-data: api/v1/process-user-data/SAVE_USER_DATA
+  remove-user-tasks: api/v1/process-user-data/REMOVE_USER_TASKS?userId={userId}
```
Comment (ghost) [inline profiles/profiles-kubernetes.yml]: same

File: achievements-manager/achievements-manager.yml [hunk 4]
```diff
@@ -73,5 +73,8 @@ health:
   config:
     enabled: false
 
+notifications:
+  url: http://notification-service
```
Comment (ghost) [inline achievements-manager/achievements-manager.yml]: should be localhost

File: gamification-manager-web/gamification-manager-web-kubernetes.yml [hunk 5]
```diff
@@ -79,3 +79,11 @@ spring:
...
         security.protocol: SSL
         session.timeout.ms: 45000
+
+events:
+  card-reply:
+    url: http://statistics/api/v1/statistics/cards/executed
+  user-profile:
+    achievements-url: http://achievements-manager/api/v1/schedule-user-achievements
+    statistics-url: http://statistics/api/v1/statistics/users
+    gamification-url: http://gamification-manager/api/v1/process-user-data
```
Comment (ghost) [inline gamification-manager-web/gamification-manager-web-kubernetes.yml]: adhere to convention

File: achievements-manager/achievements-manager-kubernetes.yml [hunk 6]
```diff
@@ -51,4 +51,8 @@ spring:
           masterAddress: "redis://qa-001.7rswex.0001.eun1.cache.amazonaws.com:6379"
           slaveAddresses:
             - "redis://qa-002.7rswex.0001.eun1.cache.amazonaws.com:6379"
-            - "redis://qa-003.7rswex.0001.eun1.cache.amazonaws.com:6379"
\ No newline at end of file
+            - "redis://qa-003.7rswex.0001.eun1.cache.amazonaws.com:6379"
+
+notifications:
+  url: http://notification-service
+  send-user-achievement: /api/v1/send-user-achievement
```
Comment (ghost) [inline achievements-manager/achievements-manager-kubernetes.yml]:
```suggestion
events:
  user-profile:
    notifications:
      url: http://notification-service
      endpoints:
        user-achievement: /api/v1/user-achievement
```

File: gamification-manager/gamification-manager.yml [hunk 7]
```diff
@@ -113,3 +113,7 @@ health:
     enabled: false
 sentry:
   dsn: https://d2c258aaf0194406a4b684d5671b8f03@o1167145.ingest.sentry.io/6257927
+
+statistics:
+  url: http://localhost:8078
+  send-executed-card: /api/v1/statistics/cards/executed
```
Comment (ghost) [inline gamification-manager/gamification-manager.yml]: events.card-reply.statistics.url
events.card-reply.statistics.endpoints.executed-card

File: api-gateway/api-gateway-kubernetes.yml [hunk 8]
```diff
@@ -12,8 +12,8 @@ spring:
         - id: profiles
           uri: http://profiles
           predicates:
-            - Path=/api/v1/profiles, /api/v1/profiles/*/*, /api/v1/therapist-profiles/**, /api/v1/therapist-profiles, /api/v1/programs/assign-program, /api/v1/profiles/api-doc
-            - Method=GET
+            - Path=/api/v1/profiles, /api/v1/profiles/*/*, /api/v1/therapist-profiles/**, /api/v1/therapist-profiles, /api/v1/programs/assign-program, /api/v1/profiles/api-doc, /api/v1/update-payment-status/{action}
```
Comment (ghost) [inline api-gateway/api-gateway-kubernetes.yml]:
```suggestion
            - Path=/api/v1/profiles/**,/api/v1/therapist-profiles/**,/api/v1/programs/assign-program/**
```

File: api-gateway/api-gateway-kubernetes.yml [hunk 9]
```diff
@@ -91,8 +91,8 @@ spring:
         - id: gamification-manager
           uri: http://gamification-manager
           predicates:
-            - Path=/api/v1/cards/**, /api/v1/tasks/**, /api/v1/categories/**, /api/v1/gamification-manager/**, /api/v1/todo/**, /api/v1/task-executions/**
-            - Method=GET
+            - Path=/api/v1/cards/**, /api/v1/tasks/**, /api/v1/categories/**, /api/v1/gamification-manager/**, /api/v1/todo/**, /api/v1/task-executions/**, /api/v1/process-user-data/**
```
Comment (ghost) [inline api-gateway/api-gateway-kubernetes.yml]:
```suggestion
            - Path=/api/v1/cards/**, /api/v1/tasks/**, /api/v1/categories/**, /api/v1/gamification-manager/**, /api/v1/todo/**, /api/v1/task-executions/**, /api/v1/user-data/**
```

File: api-gateway/api-gateway-kubernetes.yml [hunk 10]
```diff
@@ -147,7 +147,7 @@ spring:
         - id: notification-service
           uri: http://notification-service
           predicates:
-            - Path=/api/v1/notifications/**, /api/v1/emails, /api/v1/notification-service/api-doc
+            - Path=/api/v1/notifications/**, /api/v1/emails, /api/v1/notification-service/api-doc, /api/v1/send-user-achievement
```
Comment (ghost) [inline api-gateway/api-gateway-kubernetes.yml]:
```suggestion
            - Path=/api/v1/notifications/**, /api/v1/emails, /api/v1/notification-service/api-doc, /api/v1/user-achievements
```

File: api-gateway/api-gateway.yml [hunk 11]
```diff
@@ -69,7 +70,10 @@ spring:
         - id: gamification-manager
           uri: http://localhost:8080
           predicates:
-            - Path=/api/v1/achievements/**, /api/v1/cards/**, /api/v1/posts/**, /api/v1/steps/**, /api/v1/tasks/**, /api/v1/gamification-manager/**
+            - Path=/api/v1/achievements/**, /api/v1/cards/**, /api/v1/posts/**, /api/v1/steps/**, /api/v1/tasks/**, /api/v1/gamification-manager/**, /api/v1/process-user-data/**
```
Comment (ghost) [inline api-gateway/api-gateway.yml]:
```suggestion
            - Path=/api/v1/achievements/**, /api/v1/cards/**, /api/v1/posts/**, /api/v1/steps/**, /api/v1/tasks/**, /api/v1/gamification-manager/**, /api/v1/user-data/**
```

File: payments-manager/payments-manager.yml [hunk 12]
```diff
@@ -55,3 +55,7 @@ health:
 
 sentry:
   dsn: https://d2c258aaf0194406a4b684d5671b8f03@o1167145.ingest.sentry.io/6257927
+
+events:
+  client-payments:
+    profiles-url: http://localhost:8989/api/v1/update-payment-status
```
Comment (ghost) [inline payments-manager/payments-manager.yml]:
```suggestion
events:
  client-payments:
    profiles:
      url: http://localhost:8989
      endpoints:
        status: /api/v1/profiles/payment-status
```

File: payments-manager/payments-manager-kubernetes.yml [hunk 13]
```diff
@@ -4,3 +4,6 @@ spring:
 nets:
   checkout-url: https://client-portal-qa.thehousehealth.com/onboarding
 
+events:
+  client-payments:
+    profiles-url: http://profiles/api/v1/update-payment-status
```
Comment (ghost) [inline payments-manager/payments-manager-kubernetes.yml]:
```suggestion
    profiles:
      url: http://profiles/api/v1/profiles/payment-status
```

File: gamification-manager-web/gamification-manager-web.yml [hunk 14]
```diff
@@ -113,3 +113,11 @@ health:
...
 sentry:
   dsn: https://d2c258aaf0194406a4b684d5671b8f03@o1167145.ingest.sentry.io/6257927
+
+events:
+  card-reply:
+    url: http://localhost:8078/api/v1/statistics/cards/executed
+  user-profile:
+    achievements-url: http://localhost:8182/api/v1/schedule-user-achievements
+    statistics-url: http://localhost:8078/api/v1/statistics/users
+    gamification-url: http://localhost:8081/api/v1/user-data
```
Comment (ghost) [inline gamification-manager-web/gamification-manager-web.yml]:
```suggestion
events:
  card-reply:
    statistics:
      url: http://localhost:8078
      endpoints:
        executed: /api/v1/statistics/cards/executed
  user-profile:
    achievements:
      url: http://localhost:8182
      endpoints:
        schedule: /api/v1/profiles/user-achievements
    statistics:
      url: http://localhost:8078
      endpoints:
        users: /api/v1/statistics/users
    gamification:
      url: http://localhost:8081
      endpoints:
        user-data: /api/v1/user-data
```

File: gamification-manager-web/gamification-manager-web-kubernetes.yml [hunk 15]
```diff
@@ -79,3 +79,11 @@ spring:
...
         security.protocol: SSL
         session.timeout.ms: 45000
+
+events:
+  card-reply:
+    url: http://statistics/api/v1/statistics/cards/executed
+  user-profile:
+    achievements-url: http://achievements-manager/api/v1/schedule-user-achievements
+    statistics-url: http://statistics/api/v1/statistics/users
+    gamification-url: http://gamification-manager/api/v1/user-data
```
Comment (ghost) [inline gamification-manager-web/gamification-manager-web-kubernetes.yml]:
```suggestion
events:
  card-reply:
    statistics:
      url:  http://statistics
      endpoints:
        executed: /api/v1/statistics/cards/executed
  user-profile:
    achievements:
      url: http://achievements-manager
      endpoints:
        schedule: /api/v1/profiles/user-achievements
    statistics:
      url:  http://statistics
      endpoints:
        users: /api/v1/statistics/users
    gamification:
      url: http://gamification-manager
      endpoints:
        user-data: /api/v1/user-data
```

File: profiles/profiles-kubernetes.yml [hunk 16]
```diff
@@ -81,3 +81,8 @@ program:
 calendly:
   organization-id: 9b1093f4-7238-45bb-82f1-cd7d3278bf04
   key: '{cipher}0d51e0bef2641c5e5be7a2bad3b29d4c9d8844de0587ab0b5b91d02428322ca905781969ff8397edc23a441a0f57803da968f0b05df1a338b82e33a94f96f31d137941ff7281ed6af6415ec3c48e2abb94518d200f2d63e28c48434497ea05b33eec4ac643897f3ea81adfaab8082667457fec4c3ad725de225d1fdcca93607021b1eaa112be909b6a615820597b20d3c95e33b79ed0c2ea1282030c491fed0efbd481471ffaeb6b854d2d73b8aceae7ef77c7f41946354484457933a3837914dac36db76a0eba72ef8ca238578a67f1e11ff63e4bb0f67c019de731d01d36fdd5b520c7751f5efb5a7dd25d532c5eb96a867228a1317c1fee712fec74918866e5172c5b95475db6c6863171be7a50f8481196afb4b773b7f535c59ec04a773989771950b01361d56e706bd46c50067e186be0fd68fddc3bcc2313fff4a88f392b57357140fa876715a238fbabf9d25c6a4638bd7c62c7153fb8e2976b121449cb865714b2096066dbb85a2d31861a90ae0e84679ed08151860b828adb4fe95d73f8cb6ecc0bec965da86fb795ef0d0e721a39818653b8d3660014b46f4695595945f63fc361d67ec0bda459efbdfef4dfa8f0a0d01d8e57105be1c09f17a4f0'
+
+gamification:
+  url: http://gamification-manager
+  save-user-data: api/v1/user-data/SAVE_USER_DATA
+  remove-user-tasks: api/v1/user-data/REMOVE_USER_TASKS?userId={userId}
```
Comment (ghost) [inline profiles/profiles-kubernetes.yml]:
```suggestion
events:
  user-profile:
    gamification:
      url: http://gamification-manager
      endpoints:
        save-user-data: /api/v1/user-data/save
        remove-user-tasks: /api/v1/user-data/tasks/remove?userId={userId}
```

File: payments-manager/payments-manager-kubernetes.yml [hunk 17]
```diff
@@ -4,3 +4,7 @@ spring:
 nets:
   checkout-url: https://client-portal-qa.thehousehealth.com/onboarding
 
+events:
+  client-payments:
+    profiles:
+      url: http://profiles/api/v1/profiles/payment-status
```
Comment (ghost) [inline payments-manager/payments-manager-kubernetes.yml]:
```suggestion
events:
  client-payments:
    profiles:
      url: http://profiles
      endpoints:
        status: /api/v1/profiles/payment-status
```

File: achievements-manager/achievements-manager-kubernetes.yml [hunk 18]
```diff
@@ -51,4 +51,11 @@ spring:
...
-            - "redis://qa-003.7rswex.0001.eun1.cache.amazonaws.com:6379"
\ No newline at end of file
+            - "redis://qa-003.7rswex.0001.eun1.cache.amazonaws.com:6379"
+
+events:
+  user-profile:
+    notifications:
+      url: http://notification-service
+      endpoints:
+        user-achievement: /api/v1/send-user-achievement
```
Comment (ghost) [inline achievements-manager/achievements-manager-kubernetes.yml]:
```suggestion
        user-achievement: /api/v1/user-achievements
```

File: achievements-manager/achievements-manager.yml [hunk 19]
```diff
@@ -73,5 +73,9 @@ health:
...
     enabled: false
 
-
-
+events:
+  user-profile:
+    notifications:
+      url: http://notification-service
+      endpoints:
+        user-achievement: /api/v1/send-user-achievement
```
Comment (ghost) [inline achievements-manager/achievements-manager.yml]:
```suggestion
        user-achievement: /api/v1/user-achievements
```

File: api-gateway/api-gateway-kubernetes.yml [hunk 20]
```diff
@@ -129,14 +129,14 @@ spring:
         - id: achievements-manager
           uri: http://achievements-manager
           predicates:
-            - Path=/api/v1/achievements/**, /api/v1/achievements-monthly/**, /api/v1/achievements-weekly/**, /api/v1/achievements-scheduled/**, /api/v1/step-user/**, /api/v1/achievements-step/**, /api/v1/achievements-step, /api/v1/achievement-user/**, /api/v1/achievements-manager/api-doc
-            - Method=GET
+            - Path=/api/v1/achievements/**, /api/v1/achievements-monthly/**, /api/v1/achievements-weekly/**, /api/v1/achievements-scheduled/**, /api/v1/step-user/**, /api/v1/achievements-step/**, /api/v1/achievements-step, /api/v1/achievement-user/**, /api/v1/achievements-manager/api-doc, /api/v1/schedule-user-achievements, /api/v1/me/step/{step}, /api/v1/me, /api/v1/me/achievements/{achievementId}/notify, /api/v1/me/achievements/schedule
```
Comment (okuzan-b11) [inline api-gateway/api-gateway-kubernetes.yml:132]:
```suggestion
            - Path=/api/v1/achievements/**, /api/v1/achievements-monthly/**, /api/v1/achievements-weekly/**, /api/v1/achievements-scheduled/**, /api/v1/step-user/**, /api/v1/achievements-step/**, /api/v1/achievements-step, /api/v1/achievement-user/**, /api/v1/achievements-manager/api-doc, /api/v1/schedule-user-achievements, /api/v1/me/step/{step}, /api/v1/me
```

Comment (okuzan-b11) [reply api-gateway/api-gateway-kubernetes.yml:132]: should be covered by api/v1/achievements/**?

Comment (ghost) [inline api-gateway/api-gateway-kubernetes.yml]:
```suggestion
            - Path=/api/v1/achievements/**, /api/v1/achievements-monthly/**, /api/v1/achievements-weekly/**, /api/v1/achievements-scheduled/**, /api/v1/step-user/**, /api/v1/achievements-step/**, /api/v1/achievements-step, /api/v1/achievement-user/**, /api/v1/achievements-manager/api-doc, /api/v1/me/**
```

File: gamification-manager-web/gamification-manager-web-kubernetes.yml [hunk 21]
```diff
@@ -79,3 +79,23 @@ spring:
         bootstrap.servers: b-1.kafkaqa.238fcd.c2.kafka.eu-north-1.amazonaws.com:9094, b-2.kafkaqa.238fcd.c2.kafka.eu-north-1.amazonaws.com:9094
         security.protocol: SSL
         session.timeout.ms: 45000
+
+events:
+  card-reply:
+    statistics:
+      url:  http://statistics
```
Comment (ghost) [inline gamification-manager-web/gamification-manager-web-kubernetes.yml]:
```suggestion
      url: http://statistics
```

File: gamification-manager-web/gamification-manager-web-kubernetes.yml [hunk 22]
```diff
@@ -79,3 +79,23 @@ spring:
...
+      endpoints:
+        executed: /api/v1/statistics/cards/executed
+  user-profile:
+    achievements:
+      url: http://achievements-manager
+      endpoints:
+        schedule: /api/v1/{userId}/achievements/schedule
+    statistics:
+      url:  http://statistics
```
Comment (ghost) [inline gamification-manager-web/gamification-manager-web-kubernetes.yml]:
```suggestion
      url: http://statistics
```

**Pull request closed without merging**